net/interfaces/interfaces_windows.go

```diff
@@ -71,3 +72,22 @@ func likelyHomeRouterIPWindows() (ret netaddr.IP, ok bool) {
 	})
 	return ret, !ret.IsZero()
 }
+
+// NonTailscaleMTUs returns a map of interface LUID to interface MTU,
+// for all interfaces except Tailscale tunnels.
+func NonTailscaleMTUs() (map[uint64]uint32, error) {
+	ifs, err := winipcfg.GetInterfaces()
+	if err != nil {
+		return nil, err
+	}
+
+	ret := map[uint64]uint32{}
+	for _, iface := range ifs {
+		if iface.Description == "Tailscale Tunnel" {
```

Comment:
still think this should be pulled out into a, say, `tsconst` package or something.

Reply:
Confused. I thought you wanted the interface selection logic to be in a single place, so there would only be one place to fix. Not convinced creating a brand new package and moving a constant there is going to help, but if you're set on it, I can make the change.

Reply:
My concern was all the possible places we might repeat the constant `"Tailscale Tunnel"` and I'm already concerned because I can't find the others now.

cmd/tailscaled and the corp/win equivalent both pass a `tunname string` of `"Tailscale"`, without the ` Tunnel` suffix.

And `wintun/` and `wireguard-...` AH! FOUND IT!

```
bradfitz@tsdev:~/src/tailscale.io/wireguard-go$ git grep " Tunnel"
tun/wintun/wintun_windows.go:   return fmt.Sprintf("%s Tunnel", pool)
```

I want to have fewer places creating and checking that magic string. I'll work on cleaning them up a bit more.